test/functional/p2p_v2_misbehaving.py

```
@@ -91,7 +90,7 @@ def complete_handshake(self, response):
 class NonEmptyVersionPacketState(EncryptedP2PState):
     """"Add option for sending non-empty transport version packet."""
     def complete_handshake(self, response):
-        self.transport_version = random_bytes(5)
+        self.transport_version = random.randbytes(5)
```

Ruff (0.15.9)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)